cocotb/handle.py

```diff
@@ -577,6 +577,26 @@ def loads(self):
         """An iterator for gathering all loads on a signal."""
         return _SimIterator(self._handle, simulator.LOADS)
 
+class SetAction:
+    """Base class representing the type of action used while setting a handle value."""
+    def __init__(self, value=None):
+        self.value = value
+
+class Deposit(SetAction):
+    """Action used for placing a value into a given handle."""
+    _action = 0
+
+class Force(SetAction):
+    """Action used to force a handle to a given value until a release is applied."""
+    _action = 1
+
+class Release(SetAction):
```

Comment:
I don't think `Release(0xFF)` should be legal if you're going to ignore the value. Can you perhaps instead have:
```python
class _SetAction: pass  # no .value member

class _SetValueAction(_SetAction):
    # what you currently have as _SetAction

class Release(_SetAction): ...

class Force(_SetValueAction): ...
```
Note also that I added a leading `_` for `SetAction` - I don't think we want users using it directly.